sdk/core/System.ServiceModel.Rest/src/Pipeline/Pipeline.cs

```diff
@@ -1,16 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System.Threading.Tasks;
-
-namespace System.ServiceModel.Rest.Core.Pipeline;
-
-// TODO: if we want this to be non-HTTP, should it not be in a "Rest" namespace?
-public abstract class Pipeline<TMessage, TMessageOptions>
```

Comment:
I considered moving it, but decided I want to keep it and just constraint TMessage to PipelineMessage

Reply:
We may still be able to remove a layer off the top of the message concept, though?